tests/features/tags/add-tag.spec.ts

```diff
@@ -0,0 +1,132 @@
+import { expect } from "@playwright/test";
+import { test } from '../../config/test-base';
+import { go_to_page } from "../../ui/steps/login";
+
+test.describe('Tags', function () {
+    let tags_all;
+    let tag_name;
+
+    test('Add new Tag', async ({steps: {pages}, page}) => {
```

Comment:
can we mention link to initial test cases?

Example:
```
...
test.describe('Tags', function () {
    /**
     * https://odd.testops.cloud/project/1/test-cases/7
     */
    test('Add new Tag', async ({steps: {pages}, page}) => {
    ...
```

Reply:
Done